common/src/test/java/com/microsoft/identity/common/unit/ThreadUtilsTests.java

```diff
@@ -0,0 +1,81 @@
+package com.microsoft.identity.common.unit;
```

Comment:
add license

Reply:
I bet there's a GitHub plugin for this...